Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocCommandColumnRenderer.cs

```diff
@@ -81,8 +81,8 @@ private void RenderCellCommand (BocColumnRenderingContext<BocCommandColumnDefini
       if (renderingContext.ColumnDefinition.Icon.HasRenderingInformation)
         renderingContext.ColumnDefinition.Icon.Render (renderingContext.Writer, renderingContext.Control);
 
-      if (!string.IsNullOrEmpty (renderingContext.ColumnDefinition.Text))
-        renderingContext.Writer.Write (renderingContext.ColumnDefinition.Text); // Do not HTML encode
+      if (!renderingContext.ColumnDefinition.Text.IsEmpty)
+        renderingContext.ColumnDefinition.Text.Write (renderingContext.Writer); // Do not HTML encode
```

Comment:
```suggestion
        renderingContext.ColumnDefinition.Text.Write (renderingContext.Writer);
```

These comments are now obsolete.